addons/portal/static/src/xml/portal_chatter.xml

```diff
@@ -114,9 +114,11 @@
 
     <!-- Chatter: internal toggle widget -->
     <t t-name="portal.chatter_internal_toggle">
-        <div t-attf-class="float-right o_portal_chatter_js_is_internal #{message.is_internal and 'o_portal_message_internal_on' or 'o_portal_message_internal_off'}"
+        <div t-attf-class="float-right o_portal_chatter_js_is_internal #{message.is_note ? 'o_portal_message_subtype_internal' :
```

Comment:
This becomes a bit complicated and hard to read, especially since there is JS code that toggles the classes "o_portal_message_internal_on" and "o_portal_message_internal_off".

Can't we instead:

```
<div t-if="message.is_note">
    -> your button here
</div>
<div t-else="" t-attf-class="..." ...> --> the current div and its content with the additional "t-else".
</div>
```